### PR TITLE
Reorder Dockerfile for better use of cache

### PR DESCRIPTION
**Pull request by rhcarvalho**
Every change to the scripts was causing yum install to run over and over.

@bparees PTAL

### CODE REVIEW

File: 5.5/Dockerfile [hunk 1]
```diff
@@ -36,6 +33,9 @@ RUN rpmkeys --import file:///etc/pki/rpm-gpg/RPM-GPG-KEY-CentOS-7 && \
     test "$(id mysql)" = "uid=27(mysql) gid=27(mysql) groups=27(mysql)" && \
     chmod o+w -R /var/lib/mysql
 
+COPY run-*.sh /usr/local/bin/
+COPY contrib /var/lib/mysql/
```
Comment (bparees) [inline 5.5/Dockerfile]: same comment as the other PR (but that chmod o+w should be a chmod a+rwx i think.  or at least a+rw)

**Comment by rhcarvalho** [comment]
@bparees and this is the last change of this type for the time being. PTAL

**Comment by bparees** [comment]
lgtm, will wait for test to finish.

**Comment by mfojtik** [comment]
LGTM (do we need to change other dockerfiles?)

**Comment by rhcarvalho** [comment]
@mfojtik I changed mongodb, postgresql, mysql. That might be it.

**Pull request merged**